components/content/ProseImg.vue

```diff
@@ -1,42 +1,62 @@
 <template>
     <span class="text-center d-block">
-        <NuxtImg :src="refinedSrc" :alt="alt" :width="width" :height="height" :class="classWithZoom" loading="lazy" format="webp" quality="80" densities="x1 x2" />
+        <NuxtImg
+            :src="refinedSrc"
+            :alt="alt"
+            :width="width"
+            :height="height"
+            :class="classWithZoom"
+            loading="lazy"
+            format="webp"
+            quality="80"
+            densities="x1 x2"
+        />
```

Comment:
I think this way is by far more readable than long lines. If you think it's a mistake please I'd like to know the arguments.